sdk/storage/Azure.Storage.DataMovement/src/CommitChunkHandler.cs

```diff
@@ -92,11 +92,12 @@ public CommitChunkHandler(
                     SingleReader = true,
                 });
             _cancellationToken = cancellationToken;
-            _processStageChunkEvents = Task.Run(() => NotifyOfPendingStageChunkEvents());
 
             // Set bytes transferred to block size because we transferred the initial block
             _bytesTransferred = blockSize;
```

Comment:
Moving this line above 

`_processStageChunkEvents = Task.Run(() => NotifyOfPendingStageChunkEvents());`

doesn't really make too much of a difference as we will virtually always initialize `_bytesTransferred`  before we can select a thread from the thread pool and kick off `NotifyOfPendingStageChunkEvents()` but I just think its easier to see this way